src/integrationTest/java/com/mongodb/hibernate/BasicCRUDTests.java

```diff
@@ -170,6 +172,46 @@ void testSimpleDeletion() {
         }
     }
 
+    @Nested
+    class UpdateTests {
```

Comment:
`@Nested` is used to accommodate the future new testing case after `optimistic locking` feature is taken into consideration, even though currently there is only one testing case in this PR.